resources/assets/icons/status/debug.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g transform="translate(1.5 1.5)" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"><circle cx="8.5" cy="8.5" r="9"/><path d="M8.5 8.5V5.2m0 3.3l3.9 3.9"/></g></svg>
```

Comment:
identical to 
![image](https://user-images.githubusercontent.com/35610748/128482241-deb9434e-cb65-47dd-a792-65107ead83d1.png)

Reply:
@ItsANameToo its not identical, this one has wider lines
Waiting restart
![image](https://user-images.githubusercontent.com/17262776/128522810-3e012569-7b43-4ede-b347-ea4c64cc756b.png)
Debug:
![image](https://user-images.githubusercontent.com/17262776/128522842-f5b8560c-cf00-47c2-9961-f9a80562557f.png)

Reply:
can you ask oleg which one to use, they have identical designs

Reply:
FYI I asked Oleg about this